ppml/trusted-big-data-ml/scala/docker-occlum/py-examples/tpch/main.py

```diff
@@ -0,0 +1,68 @@
+#
```

Comment:
Can we move TPCH into zoo-tutorial? In that case, both occlum and Gramine can use it by adding it into docker image?

Reply:
Good idea， i will do it.